pkg/cli/cmds/executions.go

```diff
@@ -114,6 +138,9 @@ func executionDelete(c *cli.Context) error {
 	}
 
 	for _, exec := range execs.Items {
+		if exec.CreationTimestamp.After(from) {
+			continue
+		}
 		err := controllers.executions.Delete(namespace, exec.Name, &metav1.DeleteOptions{})
 		logrus.Info("deleting " + exec.Name)
```

Comment:
nit: could probably put this above delete

Reply:
fixing